proposals/rcp-51-dom-increment.md

```diff
@@ -0,0 +1,324 @@
+# RESO DOM Increment Endorsement
+
+| **RCP** | 51 |
+| :--- | :--- |
+| **Version** | **1.0.0** |
+| **Authors** | Bryan Burgers, Zenlist<br />Al McElmon, CoreLogic<br />Sergio Del Rio, T4BI |
+| **Specification** | [**LINK TO RCP**](#) |
+| **Status** | IN PROGRESS |
+| **Date Ratified** | Month YYYY |
+| **Dependencies** | [Data Dictionary][RCP-40] |
```

Comment:
This should be RCP-36, with a link to the SHA for the 2.0 version.

https://github.com/RESOStandards/transport/blob/a99fb6ca307208280ac51bca1f573e89cb67b202/proposals/data-dictionary.md

Reply:
Done.